app/internal/notes/notes.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+from typing import Any, Dict, List
+
+from fastapi import HTTPException, status
+from sqlalchemy.orm import Session
+
+from app.database.models import Note
+from app.database.schemas import NoteSchema
+
+
+async def create(session: Session, payload: NoteSchema) -> int:
+    note = Note(title=payload.title, description=payload.description)
+    session.add(note)
+    session.commit()
+    session.refresh(note)
+    return note.id
+
+
+async def view(session: Session, id: int) -> Note:
```

Comment:
pretty sure you got a warning for using "id" as a variable name here.

Reply:
Done